compiler/intf.ml

```diff
@@ -309,7 +309,7 @@ let emit_callback_wrapper oc intf meth =
 (* Declare external callback wrapper *)
 
 let declare_callback_wrapper oc intf meth =
-  let (ins, outs) = ml_view meth in
+  let (_ins, _outs) = ml_view meth in
   (* Emit function header *)
```

Comment:
Since `ml_view` is a pure function and `ins` and `out` are not used, let's just remove this "let".